refactron/cli/run.py

```
@@ -0,0 +1,41 @@
+    console.print()
+    _auth_banner("Pipeline Run")
+    console.print()
+
+    target_path = _validate_path(target) if target else Path.cwd()
+    
+    with console.status("[primary]Executing pipeline run...[/primary]"):
+        try:
+            pipeline = RefactronPipeline()
+            result = pipeline.analyze(target_path, use_incremental=incremental)
+            
+            console.print(f"[green]Pipeline analysis completed successfully.[/green]")
```

Ruff (0.15.10)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)